developer_manual/digging_deeper/api.rst

```diff
@@ -7,3 +7,23 @@ PHP public API
 
 The public API is contained in the OCP namespace. See the `OCP API reference
 <https://nextcloud-server.netlify.app/>`_ for further details.
+
+
+PHP testing API
```

Comment:
This sounds like this is about API for testing not API that is in testing state.
Maybe "Unstable PHP API"? As "unstable" is a more common term for this kind of API in general?

 (At least for me "testing" is more like we provide our `\Test` namespace)

Reply:
Was wondering similarly, also for the `lib/testing/` folder in Server. Other options could be Experimental or Preview (colliding with previews/thumbnails)

Reply:
Totally agree on those points, but from previous vocal discussions:

- `testing` is above `unstable` (at least from debian's point of view)
- 'Experimental' would means NCE to be the name of the namespace, which is an already used acronym,
- 'Preview' would also conflict,
- 'devel' can be confusing too, as it can be interpreted as API for development ...

unstable might the only valid term that does not conflict that much and also reflect the 'unstable'/'prone to changes'

Reply:
So, after discussion with @provokateurin, `testing` have been renamed to `unstable`.

Files location:  `/lib/unstable/`
namespace: `NCU\`